doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -333,7 +333,7 @@ OSD and run the following command:
 
        ceph-bluestore-tool \
         --path <data path> \
-        --sharding="m(3) p(3,0-12) o(3,0-13)=block_cache={type=binned_lru} l p" \
+        --sharding="m(3) p(3,0-12) o(3,0-13)=block_cache={type=binned_lru} L P" \
```

Comment:
As I noted in the parent tracker, the mainline fix is still missing o -> O.

Reply:
@baergj I'm confused: https://github.com/ceph/ceph/pull/54339#issuecomment-1793344421

Should I cherry-pick this to `p`, `q`, `r` releases?

Reply:
It looks wrong to me in all of those, so I'm also confused by that comment.